account_credit_control/views/credit_control_line.xml

```diff
@@ -0,0 +1,178 @@
+                            domain="[('state', 'in', ('error', 'email_error'))]"
+                            help="An error has occured during the sending of the email."/>
+                    <filter name="filter_manual" icon="fa-hand-paper-o"
+                            string="Manual change"
+                            domain="[('manually_overridden', '=', True)]"
+                            help="The line was deprecated by a manual change of policy on invoice."/>
+                    <filter name="filter_manual" icon="fa-hand-paper-o"
+                            string="Manual followup"
+                            domain="[('manually_followup', '=', True)]"
+                            help="The partner had a manual followup."/>
+                </group>
+                <group expand="0" string="Group By" name="group_by">
+                    <filter domain='[]' context="{'group_by': 'date'}"
+                            icon="terp-go-month" string="Run date"/>
+                    <filter domain='[]' context="{'group_by': 'level'}"
+                            icon="terp-gtk-jump-to-rtl" string="Level"/>
+                    <filter domain='[]' context="{'group_by': 'partner_id'}"
+                            icon="terp-partner" string="Partner"/>
+                    <filter domain='[]' context="{'group_by': 'account_id'}"
+                            icon="terp-folder-green" string="Account"/>
+                    <filter domain='[]' context="{'group_by': 'invoice_id'}"
+                            icon="terp-document-new" string="Invoice"/>
+                    <filter domain='[]' context="{'group_by': 'policy_id'}"
+                            icon="terp-document-new" string="Credit policy"/>
+                    <filter domain='[]'
+                            context="{'group_by': 'policy_level_id'}"
+                            icon="terp-document-new"
+                            string="Credit policy level"/>
+                    <filter domain='[]' context="{'group_by': 'channel'}"
+                            icon="terp-document-new" string="Channel"/>
+                    <filter domain='[]'
+                            context="{'group_by': 'manually_overridden'}"
+                            icon="terp-document-new" string="Manual change"/>
+                    <filter domain='[]'
+                            context="{'group_by': 'manual_followup'}"
+                            icon="terp-document-new" string="Manual followup"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="credit_control_line_tree" model="ir.ui.view">
+        <field name="name">credit.control.line.tree</field>
+        <field name="model">credit.control.line</field>
+        <field name="arch" type="xml">
+            <tree editable="bottom" string="Control Credit Lines"
+                  colors="green:state == 'sent';red:state in ('error', 'email_error');">
```

Comment:
The `colors="..."` syntax is no longer working in Odoo 11. Please use the `decoration-` syntax.

Reply:
Was it working in v10? We can add this to the migration guide (https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0)

Reply:
I didn't check whether is working on V10, but the `decoration-` mechanism replaces the `colors=` mechanism in the standard Odoo V10.
Maybe is better to add it in both the migration guides: Migration-to-version-10.0 and Migration-to-version-11.0.

Reply:
Can you please elaborate the text with the details as I don't know them? And thanks for the info! TIL

Reply:
@pedrobaeza we may get some inspiration from the description of this PR: https://github.com/OCA/web/pull/839